benchmark/src/environments/acpt/context/ndv4-topo.xml

```diff
@@ -0,0 +1,35 @@
+<!-- This topology file was copied from https://github.com/Azure/azhpc-images/blob/master/common/network-tuning.sh -->
```

Comment:
this file should not be necessary in ACPT, worth removing?

(not sure though, feel free to discard this comment)